v2/pkg/genruntime/extensions/error_classifier.go

```diff
@@ -42,11 +42,18 @@ func CreateErrorClassifier(
 	}
 
 	return func(cloudError *genericarmclient.CloudError) (core.CloudErrorDetails, error) {
-		log.V(Info).Info(
-			"Classifying cloud error",
-			"Message", cloudError.InnerError.Message,
-			"Code", cloudError.InnerError.Code,
-			"Target", cloudError.InnerError.Target)
+		if cloudError.InnerError != nil {
```

Comment:
Maybe just add a `String()` method to `CloudError` type and do this there?

Reply:
The changes from PR #2112 simplified this.